lib/protocol/elementIdAttribute.js

```diff
@@ -19,7 +19,13 @@ let elementIdAttribute = function (id, attributeName) {
         throw new ProtocolError('number or type of arguments don\'t agree with elementIdAttribute protocol command')
     }
 
-    return this.requestHandler.create(`/session/:sessionId/element/${id}/attribute/${attributeName}`)
+    var command = '/attribute/'
+
+    if (this.options.useW3C && this.desiredCapabilities.browserName === 'firefox' && attributeName === 'value') {
```

Comment:
Why limiting it to firefox only? If this is the new protocol we should make it for all browser. If someone has useW3C enabled we should assume the driver has the webdriver protocol implemented